src/components/StudyPane/Toolbar/Buttons.tsx

```diff
@@ -400,7 +400,11 @@ export const StructureBtn = ({
   }
   return (
     <div>
-      <button className="bg-white hover:bg-gray-100 text-gray-800 font-semibold py-2 px-4 border border-gray-400 rounded shadow"
+      <button className="text-gray-800 border-gray-400 font-semibold 
+        py-1 px-2 border border-gray-400 rounded shadow cursor-not-allowed
```

Comment:
For the grey out button, we want the background color of the button in white and the font color becomes gray.

Reply:
the font is already grey, it will be kind of hard to tell the differences.